tests/Unit/Converter/JsonConverterTest.php

```diff
@@ -1190,4 +1190,27 @@ public function convertPropertyDefinitionConvertsArrayToObjectDataProvider()
 
         return $data;
     }
+
+    public function testConvertPropertyDefinitionIgnoresUnrecognizedProperty()
+    {
+      $propertyDefinitions = $this->getResponseFixtureContentAsArray(
+          'Cmis/v1.1/BrowserBinding/getTypeDefinitionHidden-response.log'
+      )['propertyDefinitions'];
+      $expectedObjects = require(__DIR__ . '/../../Fixtures/Php/PropertyDefinitionsFixture.php');
+
+      $extensions = [new CmisExtensionElement(
+          null,
+          'isHidden',
```

Comment:
I would rename this property to a more speaking name that makes clear that this is a "custom property". Something like `customPropertyThatIsNotPartOfCmis` instead of `isHidden`. Makes sense?